front_end_pocket_poll/lib/events_widgets/event_details_closed.dart

```diff
@@ -196,8 +195,8 @@ class _EventDetailsClosedState extends State<EventDetailsClosed> {
     );
   }
 
-  void getEvent() {
-    this.event = Globals.currentGroupResponse.group.events[widget.eventId];
+  void getEvent(final Event event) {
```

Comment:
I think this should be renamed now to something like "buildUserRows". I was very confused what this method was doing at first by calling it getEvent.

Reply:
Oh right, I was just using what was already there with no thought to what it was actually doing after the update. I updated it to buildUserRows as suggested.